util/topgen.py

```diff
@@ -1009,9 +1030,13 @@ def main():
     hjson_dir = Path(args.topcfg).parent
     genhjson_path = hjson_dir / ("autogen/top_%s.gen.hjson" %
                                  completecfg["name"])
-    gencmd = (
-        "// util/topgen.py -t hw/top_{topname}/data/top_{topname}.hjson --hjson-only "
-        "-o hw/top_{topname}/\n".format(topname=topname))
+
+    # Header for HJSON
+    gencmd = '''// util/topgen.py -t hw/top_{topname}/data/top_{topname}.hjson \\
```

Comment:
tiny nit, the alignment of the contents in quotation is a different from the one done later.  I think the one later is a bit neater.

Reply:
oh this is because it is a heredoc string. it should print out correctly aligned. compared to the SV header below there is however a missing comment line at the beginning. I fixed that.